tests/Aspire.Hosting.SqlServer.Tests/SqlServerFunctionalTests.cs

```diff
@@ -132,7 +136,7 @@ public async Task WithDataShouldPersistStateBetweenUsages(bool useVolume)
             using var builder1 = TestDistributedApplicationBuilder.Create(o => { }, testOutputHelper);
 
             var sqlserver1 = builder1.AddSqlServer("sqlserver");
-            var masterdb1 = sqlserver1.AddDatabase("master");
+            var db1 = sqlserver1.AddDatabase(databaseName);
```

Comment:
Not using `master` updates this test to ensure the database is created. `master` was used since it's a default one in sql server.